documentation/docs/components/Cognite3DViewerDemo.tsx

```diff
@@ -23,31 +23,34 @@ export default function Cognite3DViewerDemo({
 }: DemoProps & OwnProps) {
   const canvasWrapperRef = useRef(null);
 
-  useEffect(() => {
```

Comment:
What is the difference here, using `useMemo` instead of `useEffect`?

Reply:
It is a more proper way of storing a variable that is "persistent" between component re-renders. In current component implementation it doesn't bring much benefits but makes viewer accessible to other hooks/html in this component, not only one `useEffect`.

Reply:
Nice! :+1:

Reply:
Okay, after some tests. Previous version seems to work faster for some reason, so reverting to it.